src/prefect/infrastructure/provisioners/cloud_run.py

```diff
@@ -191,44 +195,93 @@ async def _create_gcp_credentials_block(
             )
             return block_doc.id
 
-    @inject_client
-    async def provision(
-        self,
-        work_pool_name: str,
-        base_job_template: dict,
-        client: Optional[PrefectClient] = None,
-    ) -> Dict[str, Any]:
-        assert client, "Client injection failed"
-        await self._verify_gcloud_ready()
-        self._project = await self._get_project()
-        self._region = await self._get_default_region()
-
-        table = Panel(
+    async def _create_provision_table(self, work_pool_name: str, client: PrefectClient):
+        return Panel(
             dedent(
                 f"""\
                     Provisioning infrastructure for your work pool [blue]{work_pool_name}[/] will require:
 
                         Updates in GCP project [blue]{self._project}[/] in region [blue]{self._region}[/]
 
                             - Activate the Cloud Run API for your project
-                            - Create a service account for managing Cloud Run jobs: [blue]prefect-cloud-run[/]
+                            - Create a service account for managing Cloud Run jobs: [blue]{self._service_account_name}[/]
                                 - Service account will be granted the following roles:
                                     - Service Account User
                                     - Cloud Run Developer
-                            - Create a key for service account [blue]prefect-cloud-run[/]
+                            - Create a key for service account [blue]{self._service_account_name}[/]
```

Comment:
For consistent punctuation
```suggestion
                            - Create a key for service account: [blue]{self._service_account_name}[/]
```